### PR TITLE
Doc format

### PR DESCRIPTION
**Pull request by maleck13**
some missing code quotes

### CODE REVIEW

File: doc/user-guides/secure-protect-connect-single-multi-cluster.md [hunk 1]
```diff
@@ -351,6 +351,15 @@ You can use `curl` to hit your endpoint. You should see a `403` Because this exa
 curl -k -w "%{http_code}" https://$(kubectl get httproute test -n ${gatewayNS} -o=jsonpath='{.spec.hostnames[0]}')
 ```
 
+
+### Opening up the Gateway for other namespaces:
+
+As you have now configured the Gateway, secured it with Kuadrant policies and tested it, you can now open it up for use by other teams in other namespaces:
+
+```bash
+kubectl patch gateway ${gatewayName} -n ${gatewayNS} --type='json' -p='[{"op": "replace", "path": "/spec/listeners/0/allowedRoutes/namespaces/from", "value":"All"}]'
```
Comment (pehala) [inline doc/user-guides/secure-protect-connect-single-multi-cluster.md:360]: Wouldnt it be better to include it at original Gateway creation section?

Comment (maleck13) [reply doc/user-guides/secure-protect-connect-single-multi-cluster.md:360]: The original gateway limits HTTPRoutes to the same ns as it is not protected. It is only at this point you have applied the policies and tested connectivity and know it is safe for other namespaces to use the Gateway. At least this is the concept it is intended to convey

**Review by jasonmadigan — APPROVED** [review]
lgtm.

I can push this to 0.7.0 docs after (a bit manual right now)

**Review by pehala — APPROVED** [review]

**Comment by eguzki** [comment]
@maleck13 do you still want to push for this?

**Pull request closed without merging**